iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ShuntCompensatorNonLinearModelImpl.java

```diff
@@ -49,19 +49,30 @@ public double getG() {
 
     @Override
     public double getMaximumB() {
-        return sections.values().stream()
-                .map(SectionImpl::getB)
-                .max(Double::compare)
-                .orElseThrow(() -> new PowsyblException("a shunt compensator must have at least one section"));
+        double sumB = sections.values().stream().mapToDouble(SectionImpl::getB).sum();
```

Comment:
I am not sure to well understand: for each section (b and g) could be of any sign ? But when a section is in service, it means that all the sections before are in service ?

Reply:
I think that the maximum has to be computed in an iterative way starting at section 1 if we don't have any assumption on the sign of b of a section.